src/types.ts

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Prettier rules below, by running the `lint` command we should be able to get rid of them, mostly errors are commas inside interfaces's properties.